File: MyCollabUI/WebContent/blogs/BlogCommentService.js
/**
 * BlogCommentService
 */

app.factory('BlogCommentService',function($http,$cookieStore,$rootScope){
	
	var blogCommentService = {}
	
	var BASE_URL = 'http://localhost:8181/MyCollab';
		
	blogCommentService.retrieveComment = function (blogId){
		
		return $http.get(BASE_URL+"/getBlogComment/"+blogId)
	}
	
	
	blogCommentService.addComment = function (comment){
		
		return $http.post(BASE_URL+"/addBlogComment",comment)
	}
	
	blogCommentService.updateComment = function (comment){
		
		return $http.post(BASE_URL+"/updateBlogComment",comment)
	}
	
	blogCommentService.deleteComment = function (comment){
		
		return $http.post(BASE_URL+"/deleteBlogComment",comment)
	}
	
	return blogCommentService;
})

File: MyCollabBackEnd/src/main/java/com/linkin/service/BlogCommentService.java
package com.linkin.service;

import java.util.List;

import com.linkin.model.BlogComment;

public interface BlogCommentService {

	public boolean addBlogComment(BlogComment comment);
	
	public boolean updateBlogComment(BlogComment comment);
	
	public boolean deleteBlogComment(BlogComment comment);
	
	public BlogComment getBlogComment(int blogCommentId);
	
	public List<BlogComment> getAllBlogComments(int blogId);
}
